pandana/network.py

```diff
@@ -455,7 +458,34 @@ def plot(
 
         return bmap, fig, ax
 
-    def set_pois(self, category, maxdist, maxitems, x_col, y_col):
+    def init_pois(self, num_categories, max_dist, max_pois):
+        """
+        Initialize the point of interest infrastructure.
+
+        Parameters
+        ----------
+        num_categories : int
+            Number of categories of POIs
+        max_dist : float
+            Maximum distance that will be tested to nearest POIs. This will
+            usually be a distance unit in meters however if you have
+            customized the impedance this could be in other
+            units such as utility or time etc.
+        max_pois :
+            Maximum number of POIs to return in the nearest query
+
+        """
+        self.num_categories = num_categories
+        self.max_dist = max_dist
+        self.max_pois = max_pois
+        warnings.warn(
+            "init_pois will be deprecated, num_categories and max_dist \
```

Comment:
Let's make this a `DeprecationWarning` with the message "Method init_pois() is no longer needed in Pandana 0.4+ and will be removed in a future version; maxdist and maxitems should now be passed to set_pois()"